pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -586,6 +589,12 @@ def newpkgcb(pkgId, name, arch):
                                                                       filelists_xml_path,
                                                                       other_xml_path)
 
+                        # avoid SRPM if defined by user
+                        if 'srpm' in self.skip:
```

Comment:
What do you think about using a SimpleNamespace for OPT_SKIP_TYPES so that we can use an identifier here instead of a string?

Reply:
as using OPT_SKIP_TYPES to iter simplenamespace won't help

Reply:
I've changed my mind on this, it's fine for now